Makefile

```diff
@@ -28,10 +28,10 @@ test_lint:
 # for more than once in Travis. Change `make test` in .travis.yml
 # to `make test_multi`.
 test_multi:
-	cd network; \
+	cd services/skipchain; \
```

Comment:
Then instead of changing the makefile we can just use a environment variable [0] in the `.travis.yml` and set up the path using that variable, this won't affect all PR where we want to change that.

[0] https://docs.travis-ci.com/user/environment-variables/